requirements-dev.txt

```diff
@@ -1,4 +1,5 @@
 qiskit
+qiskit-aer>=0.11
```

Comment:
While aer is included in the metapackage you shouldn't do this, it's possible to cause a conflict with `qiskit` as `qiskit` pins aer to a fixed version. I would change `qiskit` to `qiskit>=0.38.0` for the time being. In a future date where aer is not in the metapackage we'll need to do this.

Reply:
done in 7dc10cdc5bdf5bf47601ba54fbb004eebcbc2a5f